src/mutations/transformAndValidateCart.js

```diff
@@ -15,6 +14,7 @@ const logCtx = { name: "cart", file: "transformAndValidateCart" };
  * @returns {undefined}
  */
 export default async function transformAndValidateCart(context, cart) {
+  const cartSchema = context.simpleSchemas.Cart;
```

Comment:
Generally we should always favor destructuring. Not sure why linting didn't catch that